charts/prefect-worker/templates/_helpers.tpl

```diff
@@ -9,15 +9,35 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Require Prefect Cloud Account ID
+*/}}
+{{- define "cloud.requiredConfig.accountId" -}}
+{{- if eq .Values.worker.apiConfig "cloud" }}
+    {{- required "A Prefect Cloud Account ID is required (worker.cloudApiConfig.accountId)" .Values.worker.cloudApiConfig.accountId | printf "%s"  -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Require Prefect Cloud Workspace ID
+*/}}
+{{- define "cloud.requiredConfig.workspaceId" -}}
+{{- if eq .Values.worker.apiConfig "cloud" }}
+    {{- required "A Prefect Cloud Workspace ID is required (worker.cloudApiConfig.workspaceId)" .Values.worker.cloudApiConfig.workspaceId | printf "%s"  -}}
```

Comment:
can you remind me why we need to pipe the value into `printf "%s"`? isn't it already a string?

Reply:
i get it for when we're constructing a bigger string w inputs, like the apiUrl below

Reply:
oh do we need the quotes?

Reply:
ok this is a good question - let me test to see if we need the `printf` pipe